app/default/application_routes.py

```diff
@@ -131,15 +130,6 @@ def tasklist(application_id):
     """
 
     application = get_application_data(application_id, as_dict=True)
-    account = get_account(account_id=application.account_id)
-    if application.status == ApplicationStatus.SUBMITTED.name:
-        with force_locale(application.language):
-            return render_template(
-                "application_submitted.html",
-                application_id=application.id,
-                application_reference=application.reference,
-                application_email=account.email,
-            )
```

Comment:
If I'm reading this right, we don't want to show the user the successful submission page anymore?

And instead just show them their dashboard?

Reply:
Oh wait.. this is on the tasklist route 🤔 

Okay I understand the fix now!  Strange.

Reply:
Here we are preventing the user going back to tasklist page when clicked on back button from submitted page, instead redirect to dashboard